whelktool/scripts/syslib/new-works-touch.groovy

```diff
@@ -0,0 +1,20 @@
+// MUST be run with --allow-loud
+
+BATCH_SIZE = 2000
+File allWorksToResave = new File(scriptDir, "new-works-touch-ids.txt")
+String touchedIds = System.getProperty("touched-ids-file", '')
+File touchedIdsFile = new File(touchedIds)
+List touched = touchedIdsFile.readLines()
+def count = 0
```

Comment:
Is this thread safe?

Reply:
Nope, need to synchronize or use an `AtomicInteger` (`incrementAndGet`)